AutoGrade/reports.py

```diff
@@ -0,0 +1,35 @@
+import statistics
+from .models import *
+import dateutil.relativedelta
+from datetime import timedelta
+
+def get_course_student_stat(course):
+    assignments = Assignment.objects.filter(course=course)
+    students = Student.objects.filter(courses=course).order_by('user__email')
+
+    course_student_data = []
+    for student in students:
+
+        student_submission_count = 0
+        completed_assignments = 0
+        total_marks_in_assignments = 0
+        total_time_taken = timedelta(seconds = 0)
+
+        for assignment in assignments:
```

Comment:
It's late so someone else needs to review these formulae =] 

@soachishti ... Do you have a few mins to go through these?

Reply:
Sure.